src/components/react-json-schemaForm/GeometryField/index.js

```diff
@@ -1,4 +1,26 @@
+import { withRouter } from 'react-router-dom';
 import { withTranslation } from 'react-i18next';
+import compose from '../../../utils/compose';
+import { connectCRUDProvider } from '../../../modules/CRUD/services/CRUDProvider';
+import { connectMapProvider } from '../../../modules/CRUD/views/Map/MapProvider';
+import { getLayersPaints } from '../../../modules/CRUD/services/CRUD';
+
 import GeometryField from './GeometryField';
 
-export default withTranslation()(GeometryField);
+const CRUDPRoviderGetter = ({
+  feature,
+  map,
+  settings,
+}) =>
+  ({
+    feature,
+    layerPaints: getLayersPaints(settings),
+    map,
+    settings,
+  });
```

Comment:
I think this would be clearer:

```suggestion
const CRUDPRoviderGetter = ({ feature, map, settings }) =>
  ({ feature, map, settings, layerPaints: getLayersPaints(settings) });

```